client/extensions/woocommerce/components/table/README.md

```diff
@@ -13,10 +13,14 @@ import Table from 'woocommerce/components/table';
 import TableRow from 'woocommerce/components/table/table-row';
 import TableItem from 'woocommerce/components/table/table-item';
 
-render: function() {
+function render() {
 	const titles = (
 		<TableRow isHeader>
-			{ [ 'One', 'Two', 'Three' ].map( ( item, i ) => <TableItem isHeader key={ i } isTitle={ 0 === i }>{ item }</TableItem> ) }
+			{ [ 'One', 'Two', 'Three' ].map( ( item, i ) => (
+				<TableItem isHeader key={ i } isTitle={ 0 === i }>
```

Comment:
I know this example uses a static array of items, but that is rarely the case. Which is why using the array keys for React element keys isn't recommended. Not a blocker though.